HLTrigger/HLTfilters/plugins/L1TTkMuonFilter.cc

```diff
@@ -105,28 +107,25 @@ bool L1TTkMuonFilter::hltFilter(edm::Event& iEvent,
   // Specific filter code
 
   // get hold of products from Event
-  Handle<l1t::TkMuonCollection> tkMuons;
+  Handle<l1t::TrackerMuonCollection> tkMuons;
   iEvent.getByToken(tkMuonToken_, tkMuons);
 
   //it looks rather slow to get the added muons back out of the filterproduct
   //so we just make a vector of passing and then add them all at the end
-  std::vector<l1t::TkMuonRef> passingMuons;
+  std::vector<l1t::TrackerMuonRef> passingMuons;
   auto atrkmuons(tkMuons->begin());
   auto otrkmuons(tkMuons->end());
-  TkMuonCollection::const_iterator itkMuon;
+  l1t::TrackerMuonCollection::const_iterator itkMuon;
   for (itkMuon = atrkmuons; itkMuon != otrkmuons; itkMuon++) {
-    double offlinePt = this->TkMuonOfflineEt(itkMuon->pt(), itkMuon->eta());
+    double offlinePt = this->TkMuonOfflineEt(itkMuon->phPt(), itkMuon->phEta());
     bool passesQual = !applyQuality_ || qualityCut_(*itkMuon);
-    if (passesQual && offlinePt >= min_Pt_ && itkMuon->eta() <= max_Eta_ && itkMuon->eta() >= min_Eta_) {
-      l1t::TkMuonRef ref(l1t::TkMuonRef(tkMuons, distance(atrkmuons, itkMuon)));
+    if (passesQual && offlinePt >= min_Pt_ && itkMuon->phEta() <= max_Eta_ && itkMuon->phEta() >= min_Eta_) {
+      l1t::TrackerMuonRef ref(l1t::TrackerMuonRef(tkMuons, distance(atrkmuons, itkMuon)));
       if (!applyDuplicateRemoval_ || !isDupMuon(ref, passingMuons)) {
         passingMuons.push_back(ref);
       }
     }
   }
-  for (const auto& muon : passingMuons) {
-    filterproduct.addObject(trigger::TriggerObjectType::TriggerL1TkMu, muon);
-  }
```

Comment:
Why is this removed?

Reply:
Shouldn't have been, it's a temporary state that was committed by mistake. But now this unearthed another problem...

Reply:
Fixed.